rules/linux/credential_access_collection_sensitive_files.toml

```
@@ -20,6 +20,37 @@ name = "Sensitive Files Compression"
 references = [
     "https://www.trendmicro.com/en_ca/research/20/l/teamtnt-now-deploying-ddos-capable-irc-bot-tntbotinger.html",
 ]
+note = """## Triage and analysis
+
+### Investigating Sensitive Files Compression
+
+The rule identifies the use of compression utility to collect known files containing sensitive information,
+such as credentials and other system configurations.
+
+#### Possible investigation steps
+
+- Identify the session entry leader and session user.
+- Examine the contents of session leading to the compression activity via the Session View.
+- Examine the command execution pattern in the session, which may lead to suspicious activities.
+- Examine the command execution post sensitive file compressions to indicate any exfiltration of sensitive file data.
+- Investigate any other alerts or malicious activities associated with the user/host during the past 48 hours.
+
+### False positive analysis
+
+- System administrators performing regular back up of sensitive files.
+- Whitelisted data backup and recovery tools executing in sensitive file paths.
+
+### Response and remediation
+
+- Initiate the incident response process based on the outcome of the triage.
+- Ensure active session(s) on the host(s) are terminated as the malicious user could exfiltrate the compressed sensitive file information.
+- Isolate the involved host(s) to prevent further post-compromise behavior.
+- If the Analysis has shown exfiltration of sensitive file data , analyse the impact of data that has been exfiltrated.
```

```suggestion
- If the Analysis has shown exfiltration of sensitive file data, analyze the impact of data that has been exfiltrated.
```
We need to move this analysis to the `#### Possible investigation steps` section